documentation/modules/exploit/linux/local/sudo_chroot_cve_2025_32463.md

```diff
@@ -0,0 +1,72 @@
+## Vulnerable Application
+
+
+Sudo before version 1.19.17p1 allows user to use `chroot` option, when executing command. The option is intended to run a command with user-selected root directory (if sudoers file allow it). Change in version 1.9.14 allows resolving paths via `chroot` using user-specified root directory when sudoers is still evaluating. This allows the attacker to trick Sudo into loading arbitrary shared object. As target shared object, Name Service Switch (NSS) operations are trigged before resolving sudoers, but after running `chroot` syscall. The module requires existing session and requires compiler on target machine (e.g. `gcc`). 
```

Comment:
Might be worthwhile to explicitly state versions 1.9.14-1.9.17p1 are vulnerable in the first sentence.  No one wants to dig to find the second bookend.  Ain't nobody got time for that(TM).